packages/core/src/utils-hoist/browser.ts

```diff
@@ -158,7 +158,7 @@ export function getLocationHref(): string {
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export function getDomElement<E = any>(selector: string): E | null {
-  if (WINDOW.document && WINDOW.document.querySelector) {
+  if (WINDOW.document && 'querySelector' in WINDOW.document) {
```

Comment:
this complains about being deprecated, which is really stupid 😬 I wonder if we need this check at all anymore 🤔 but probably...

Reply:
https://caniuse.com/queryselector seems to be supported in all the environments we say we support, I think it's fine.

Reply:
I will remove/deprecate this in a separate PR!